src/test/java/com/volunteer/enterprise/EnterpriseApplicationTests.java

```diff
@@ -10,7 +11,7 @@
 class EnterpriseApplicationTests {
 
     @Test
-    void confirmJonSnow_outputsJonSnow() {
+    void confirmVolunteerJonSnow_outputsVolunteerJonSnow() {
```

Comment:
Renamed `Volunteer` unit test to easily differentiate it against the `Company` unit test by name.